Training24/game.h

```diff
@@ -0,0 +1,27 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// game.h
+// Program on A3 branch.
+// Convert the given decimal number to different forms (Decimal, Hexadecimal, Binary) 
+// ------------------------------------------------------------------------------------------------
+
+#ifndef GAME_H
+#define GAME_H
+#define BITS 32
+#define HEX_DIGIT BITS/4
+
+/// <summary>Function to convert Decimal to Binary.</summary>
+char* Dec2Bin (int n);
+
+/// <summary>Function to convert Decimal to Hexadecimal.</summary>
+char* Dec2Hex (int h);
+
+/// <summary>Test cases for Decimal to Binary.</summary>
+void TestDec2Bin ();

```

Comment:
If it good that you've made these library functions, but test functions do not need to be part of this file as they will not be accessed by users. These are so that you know that the program works and that the [changes you've made do not break functionality](https://en.wikipedia.org/wiki/Regression_testing).